app/src/main/java/ch/epfl/cs311/wanderwave/ui/components/MiniPlayer.kt

```diff
@@ -0,0 +1,78 @@
+package ch.epfl.cs311.wanderwave.ui.components
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.size
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.LinearProgressIndicator
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.unit.dp
+import ch.epfl.cs311.wanderwave.R
+import ch.epfl.cs311.wanderwave.ui.components.animated.ScrollingTitle
+
+@Composable
+fun MiniPlayer(
+    isPlaying: Boolean,
+    onTitleClick: () -> Unit,
+    onPlayClick: () -> Unit,
+    onPauseClick: () -> Unit,
+    progress: Float
+) {
+  Column(modifier = Modifier.testTag("miniPlayer")) {
+    Row(
+        verticalAlignment = Alignment.CenterVertically,
+        horizontalArrangement = Arrangement.SpaceBetween,
+        modifier =
+            Modifier.height(60.dp)
+                .fillMaxWidth()
+                .background(MaterialTheme.colorScheme.background)) {
+          Box(modifier = Modifier.weight(1f).background(Color.Black)) {}
+
+          Column(
+              horizontalAlignment = Alignment.CenterHorizontally,
+              modifier =
+                  Modifier.weight(4f)
+                      .clickable { onTitleClick() }
+                      .testTag("miniPlayerTitleButton")) {
+                ScrollingTitle(title = "Betty (Get Money)", isPlaying = isPlaying)
+              }
+
+          IconButton(
+              modifier = Modifier.weight(1f).testTag("playPauseButton"),
+              onClick = {
+                if (!isPlaying) {
+                  onPlayClick()
+                } else {
+                  onPauseClick()
+                }
+              }) {
+                Icon(
+                    painter =
+                        if (isPlaying) painterResource(id = R.drawable.pause_icon)
+                        else painterResource(id = R.drawable.play_icon),
+                    contentDescription = "Play Pause Icons",
+                    tint = if (isPlaying) Color(0xFF1DB954) else MaterialTheme.colorScheme.primary,
```

Comment:
I think it might be beneficial to have all colors defined in a specific file (as was done for the bootcamp), to make sure we can propagate changes quickly and easily